pysat/_instrument.py

```diff
@@ -409,11 +409,17 @@ def __init__(self, platform=None, name=None, tag='', inst_id='',
         # Check to make sure value is reasonable
         if self.file_format is not None:
             # Check if it is an iterable string
-            if(not isinstance(self.file_format, str)
-               or (self.file_format.find("{") < 0)
-               or (self.file_format.find("}") < 0)):
+            if isinstance(self.file_format, str):
+                if any([self.file_format.find("{") < 0,
+                        self.file_format.find("}") < 0]):
+                    raise ValueError(''.join(['file format set to default, ',
+                                              'supplied string must be ',
+                                              'iterable [{',
+                                              self.file_format, '}]']))
```

Comment:
```suggestion
                    raise ValueError(''.join(['file format must be an ',
                                              'iterable string with key formatting [{',
                                              self.file_format, '}]']))
```
Or something that makes more sense